openshift_scalability/config/pyconfigMasterVertScalePause.yaml

```diff
@@ -1,6 +1,7 @@
 projects:
   - num: 3
     basename: c
+    node_selector_label: "computenode=true"
```

Comment:
Now that the PR to allow empty node selector got merged, we can set this to standard worker node label.

Reply:
Can we not use this `node_selector_label` name?